perun-core/src/main/java/cz/metacentrum/perun/core/api/AuthzResolver.java

```diff
@@ -24,6 +25,33 @@
 
 public class AuthzResolver {
 
+	/**
+	 * Check if the principal is authorized.
+	 *
+	 * @param sess perunSession which contains the principal.
+	 * @param policyDefinition of policy which contains authorization rules.
+	 * @param objects on which will be authorization provided.
+	 * @return true if the principal has particular rights, false otherwise.
+	 * @throws PolicyNotExistsException when the given policyDefinition does not exist in the PerunPoliciesContainer.
+	 */
+	public static boolean authorized(PerunSession sess, String policyDefinition, PerunBean... objects) throws PolicyNotExistsException {
```

Comment:
If I understand this correctly, in the Entry, you have to always specify the policy definition? Do you want to use different policies in different places in Entry? Or is it only because of tests? If so, I think it would make sense to create an overloaded version of this method where you use some default value.

Reply:
Yes, I do want to use different policies in different places in Entry so it is ok.